client/app/app.js

```diff
@@ -0,0 +1,51 @@
+import { h, Component } from 'preact'
```

Comment:
I handle all JS files as JSX files in Sublime. I'd prefer to stay with the standard .js, but I'm not closed to the other option. Sure of that ?